Remove & Reorganize Old BizOps Sections

rbradleyhaas opened the pull request:
<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

rbradleyhaas committed:
Update BizOps Sections

rbradleyhaas committed:
Delete vpmom.md

rbradleyhaas committed:
Update salesforce-administration.md

rbradleyhaas committed:
Delete sales-operations&order-admin.md

Commit:
move files around

rbradleyhaas committed:
Delete ops_ownership.png

rbradleyhaas committed:
Delete bizops_image.png

Commit:
fixing references

jasonblais commented:
@justinegeffen We may want to add redirects for the moving pages, in case the existing pages are linked to from somewhere.

justinegeffen commented:
Thanks, @jasonblais, good point! I'll sort that out.

justinegeffen committed:
Update sales.md

Minor review and changed Self Serve to "self-serve" based on how we refer to it internally.
